src/index.html

```diff
@@ -3,12 +3,44 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Enclosures</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Enclosures</h1>
+  <div class="big box" id="qa-block">
+    <div class="medium box">
+      <div class="small box">
+        <p>
+          In programming languages, a closure is a technique for implementing lexically
+          scoped name binding in a language with first-class functions. Operationally, a
+          closure is a record storing a function together with an environment.
+        </p>
+      </div>
+    </div>
```

Comment:
that are not good options for names
`outer-box`, `middle-box`, `inner-box` are better